fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4

```diff
@@ -418,6 +418,7 @@ PROPERTY: 'PROPERTY';
 QUANTILE_STATE: 'QUANTILE_STATE';
 QUANTILE_UNION: 'QUANTILE_UNION';
 QUERY: 'QUERY';
+QUEUED: 'QUEUED';
```

Comment:
`queued` should add to non-reserved keyword list

Reply:
added